### PR TITLE
Add myself GitHub username to the list of students

### LINKED ISSUE #31: Add myself (@AlexandreIorio) to the list of students
I would like to add myself to the list of students in the GitHub organization README.

### PR DESCRIPTION
**Pull request by AlexandreIorio**

### CODE REVIEW

**Comment by hadrylouis** [comment]
Could you please sign your previous commits or set your PR as Draft if you still have problems signing them

**Comment by ludelafo** [comment]
I remember we talked about this during the course and I remember we were able to fix it as well, I guess it will be easy to fix this PR.

You can now checkout the documentation; we did update it to include how to resign your commits: <https://github.com/heig-vd-dai-course/heig-vd-dai-course/blob/main/03-git-github-and-markdown/COURSE_MATERIAL.md#make-changes-if-needed>.

Regarding the discussion we had in class about the signing of the commits with a public/private key, I was able to find an answer on why we can set either a public key or a private key with Git: <https://stackoverflow.com/questions/75046499/what-is-the-internal-working-of-git-signing-commits-and-the-user-signingkey-opti>.

**TL;DR**: The private key is always used to sign the commits. If the public key is set, it will be used to identify the associated private key in the SSH agent.

Just to be clear: your private key is **never** shared or leaked when you sign your commits. Your private key is used to sign a message that is associated to the commit. The public key set on GitHub is used to decrypt the message signed with your private key. It can then validate your are the legitimate owner of the commits.

**Comment by AlexandreIorio** [comment]
I think everything is ok now

File: profile/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# HEIG-VD DAI Course
+# This change is made on the branch 'add-myself-github-username-to-the-list-of-students'
```
Comment (hadrylouis) [inline profile/README.md]: Could you please put back the old title : HEIG-VD DAI Course 
Thanks :)

**Review by hadrylouis — CHANGES_REQUESTED** [review]

**Comment by hadrylouis** [comment]
Also, could you please add a comment with the link to your issue with #xxx
You can follow the [create a pull request](https://github.com/heig-vd-dai-course/heig-vd-dai-course/blob/main/03-git-github-and-markdown/COURSE_MATERIAL.md#create-a-pull-request) section of the course material

**Comment by AlexandreIorio** [comment]
here is the link [Issue #31](https://github.com/heig-vd-dai-course/.github/issues/31)

**Review by ludelafo — APPROVED** [review]
